### PR TITLE
remove TODO and unused code

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/86763
ref:https://github.com/kubernetes/kubernetes/issues/4566

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Comment by tanjunchen** [comment]
see https://github.com/kubernetes/kubernetes/issues/4566#issuecomment-81886206
/priority backlog

File: test/e2e/framework/pod/wait.go [hunk 1]
```diff
@@ -53,14 +53,10 @@ const (
 	podScheduledBeforeTimeout = podListTimeout + (20 * time.Second)
 
 	// podStartTimeout is how long to wait for the pod to be started.
-	// Initial pod start can be delayed O(minutes) by slow docker pulls.
-	// TODO: Make this 30 seconds once #4566 is resolved.
-	podStartTimeout = 5 * time.Minute
+	podStartTimeout = 30 * time.Second
```
Comment (tanjunchen) [inline test/e2e/framework/pod/wait.go]: (30 * time.Second) this seems will result in timeout now

**Comment by neolit123** [comment]
needs gofmt update:
```

gofmt expand_less | 18s
-- | --
diff -u ./test/e2e/framework/pod/wait.go.orig ./test/e2e/framework/pod/wait.go --- ./test/e2e/framework/pod/wait.go.orig	2020-02-08 09:24:36.506235604 +0000 +++ ./test/e2e/framework/pod/wait.go	2020-02-08 09:24:36.506235604 +0000 @@ -56,7 +56,7 @@  	podStartTimeout = 30 * time.Second    	// poll is how often to poll pods, nodes and claims. -	poll             = 2 * time.Second +	poll = 2 * time.Second

```

File: test/e2e/framework/pod/wait.go [hunk 2]
```diff
@@ -53,14 +53,10 @@ const (
...
 
 	// poll is how often to poll pods, nodes and claims.
 	poll             = 2 * time.Second
-	pollShortTimeout = 1 * time.Minute
-	pollLongTimeout  = 5 * time.Minute
```
Comment (neolit123) [inline test/e2e/framework/pod/wait.go]: are these constants unused?

File: test/e2e/framework/pod/wait.go (hunk 1 above)
Comment (neolit123) [reply test/e2e/framework/pod/wait.go]: my suggestion will be to remove the TODO
```
// Initial pod start can be delayed O(minutes) by slow docker pulls.
// TODO: Make this 30 seconds once #4566 is resolved.
```

and keep the podStartTimeout to 5 minutes.
this constant seems to be used in a lot of places, so let's not touch it for now.
[1]

File: test/e2e/framework/util.go [hunk 3]
```diff
@@ -87,9 +87,7 @@ const (
 	// PodListTimeout is how long to wait for the pod to be listable.
 	PodListTimeout = time.Minute
 	// PodStartTimeout is how long to wait for the pod to be started.
-	// Initial pod start can be delayed O(minutes) by slow docker pulls.
-	// TODO: Make this 30 seconds once #4566 is resolved.
```
Comment (neolit123) [inline test/e2e/framework/util.go]: [1]

File: test/e2e/framework/pod/wait.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/framework/pod/wait.go]: yes

File: test/e2e/framework/pod/wait.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/framework/pod/wait.go]: get it . thanks

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**